packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -116,19 +117,24 @@ export function Comments( {
 		);
 	}
 
-	return threads.map( ( thread ) => (
-		<Thread
-			key={ thread.id }
-			thread={ thread }
-			onAddReply={ onAddReply }
-			onCommentDelete={ handleDelete }
-			onEditComment={ onEditComment }
-			isSelected={ selectedThread === thread.id }
-			setSelectedThread={ setSelectedThread }
-			setShowCommentBoard={ setShowCommentBoard }
-			commentSidebarRef={ commentSidebarRef }
-		/>
-	) );
+	return (
+		<VStack spacing="3">
+			<Text>{ __( 'Only logged in users can see Notes' ) }</Text>
```

Comment:
```suggestion
			<Text as="p" variant="muted">
				{ __( 'Only logged in users can see Notes.' ) }
			</Text>
```

Let's implement the design accurately.

Reply:
@t-hamano - Updated exactly as mentioned.